### ISSUE TITLE
Placeholder API seems to not work

### ISSUE THREAD

**Issue by Gimmoh** [Ktar5/Info-Board#147]
After I got answered on my request for the number of players on a bungee server, I tried it and I can't get it to work, any tips?

**Comment by pixar02** [comment]
Could you send the Bungee config and the board.yml files? That way I have an idea of what it should become.

**Comment by Gimmoh** [comment]
Hi, sure!

Bungee Config:
forge_support: false
connection_throttle_limit: 3
online_mode: false
log_commands: false
listeners:
- query_port: 25592
  motd: '&1Another Bungee server'
  priorities:
  - Hub
  bind_local_address: true
  tab_list: GLOBAL_PING
  query_enabled: false
  host: 0.0.0.0:25592
  forced_hosts:
    pvp.md-5.net: pvp
  max_players: 150
  tab_size: 60
  ping_passthrough: false
  force_default_server: true
  proxy_protocol: false
connection_throttle: 4000
groups:
  Gimmoh:
  - admin
log_pings: true
ip_forward: true
prevent_proxy_connections: false
network_compression_threshold: 256
stats: 804a3847-5c1c-4caf-abff-5f477bec67f2
disabled_commands:
- disabledcommandhere
timeout: 30000
permissions:
  default:
  - bungeecord.command.server
  - bungeecord.command.list
  admin:
  - bungeecord.command.alert
  - bungeecord.command.end
  - bungeecord.command.ip
  - bungeecord.command.reload
servers:
  Hub:
    motd: '&2&lMineLeak &8&l>> &c&lMANUTENZIONE IN CORSO'
    address: localhost:25565
    restricted: false
  Practice:
    motd: '&2&lMineLeak &8&l>> &c&lMANUTENZIONE IN CORSO'
    address: localhost:25566
    restricted: false
  BedWars:
    motd: '&2&lMineLeak &8&l>> &c&lMANUTENZIONE IN CORSO'
    address: localhost:25569
    restricted: false
player_limit: 150

--------------------------------------
Board.yml:

(The green stuff is in the config, just not here for saving space)

Info Board:
  '1':
    Show Time: 20
    global:
      default:
        Title: '&2MineLeak'
        Rows:
        - '&e&lNick'
        - '&7<player>'
        - ' '
        - '&9&lRank'
        - '&7<rank>'
        - ' '
        - '&c&lOnline'
        - '&7Totale: %bungee_total%'
        - '&7Hub: <onlineplayers>'
        - '&7Practice: %<bungee_practice>%'
        - '&7BedWars: %bungee_bedwars%'

**Comment by Miyasha** [comment]
How do I make the ranks work? I just have the code highlighted, and the rank does not write!

**Comment by Gimmoh** [comment]
Please don't ask here, we are talking about a different issue.

**Comment by pixar02** [comment]
It looks like your BedWars placeholder is correct.
What might be is capital letters. So instead of: %bungee_bedwars% it should be: %bungee_BedWars%
So for the other servers:
Practice: %bungee_Practice%
Hub: %bungee_Hub%
BedWars: %bungee_BedWars%

**Comment by Gimmoh** [comment]
Nope, stil not working, even tried different variations of this variable, but nothing works, Does anybody know who made this eCloud extension? A way to contact him? Or someone that could make this extension starting from scratch?

**Comment by pixar02** [comment]
This is the link to the Github Page of that Expansion.
https://github.com/PlaceholderAPI/Bungeecord-Expansion
What might be happening is that the expansion has to be in the server running with my plugin, but this plugin does not work on bungee. (Might do in the very future)
